packages/react-components/react-tag-picker-preview/src/utils/useInputTriggerSlot.ts

```diff
@@ -83,88 +68,90 @@ export function useInputTriggerSlot(
-    // update uncontrolled value
-    setValue(inputValue);
-
-    // handle updating active option based on input
-    const matchingOption = getOptionFromInput(inputValue);
-
-    // clear selection for single-select if the input value no longer matches the selection
-    if (!multiselect && selectedOptions.length === 1 && (inputValue.length < 1 || !matchingOption)) {
-      clearSelection(ev);
-    }
-  };
-
-  const trigger = useTriggerSlot(triggerFromProps, ref, {
-    state: options.state,
-    defaultProps,
-    elementType: 'input',
-    activeDescendantController,
-  });
-
-  trigger.onChange = mergeCallbacks(trigger.onChange, onChange);
-  trigger.onBlur = mergeCallbacks(trigger.onBlur, onBlur);
-
-  // NVDA and JAWS have bugs that suppress reading the input value text when aria-activedescendant is set
-  // To prevent this, we clear the HTML attribute (but save the state) when a user presses left/right arrows
-  // ref: https://github.com/microsoft/fluentui/issues/26359#issuecomment-1397759888
-  const [hideActiveDescendant, setHideActiveDescendant] = React.useState(false);
   // save the typing vs. navigating options state, as the space key should behave differently in each case
   // we do not want to update the combobox when this changes, just save the value between renders
   const isTyping = React.useRef(false);
 
-  /**
-   * Freeform combobox should not select
-   */
-  const defaultOnKeyDown = trigger.onKeyDown;
-  const onKeyDown = useEventCallback((ev: React.KeyboardEvent<HTMLInputElement>) => {
-    if (!open && getDropdownActionFromKey(ev) === 'Type') {
-      setOpen(ev, true);
-    }
-
-    // clear activedescendant when moving the text insertion cursor
-    if (ev.key === ArrowLeft || ev.key === ArrowRight) {
-      setHideActiveDescendant(true);
-    } else {
-      setHideActiveDescendant(false);
-    }
-
-    // update typing state to true if the user is typing
-    const action = getDropdownActionFromKey(ev, { open, multiselect });
-    if (action === 'Type') {
-      isTyping.current = true;
-    }
-    // otherwise, update the typing state to false if opening or navigating dropdown options
-    // other actions, like closing the dropdown, should not impact typing state.
-    else if (
-      (action === 'Open' && ev.key !== ' ') ||
-      action === 'Next' ||
-      action === 'Previous' ||
-      action === 'First' ||
-      action === 'Last' ||
-      action === 'PageUp' ||
-      action === 'PageDown'
-    ) {
-      isTyping.current = false;
-    }
-
-    // allow space to insert a character if freeform & the last action was typing, or if the popup is closed
-    if ((isTyping.current || !open) && ev.key === ' ') {
-      triggerFromProps?.onKeyDown?.(ev);
-      return;
-    }
-
-    defaultOnKeyDown?.(ev);
-  });
-
-  trigger.onKeyDown = onKeyDown;
-
-  if (hideActiveDescendant) {
-    trigger['aria-activedescendant'] = undefined;
-  }
-
+  const trigger = useTriggerSlot(
```

Comment:
how much of this has changed? the intention was to eventually re-use the combobox logic

Reply:
The previous code simply didn't work. It was a merge between the new `activedescendant` logic and an old local one that was basically ignored. I'll see to update `combobox` logic with whatever I change here. The final goal is definitely to have them both equal